src/sql/workbench/contrib/query/browser/queryResultsEditor.ts

```diff
@@ -76,8 +76,8 @@ export class QueryResultsEditor extends EditorPane {
 	public static ID: string = 'workbench.editor.queryResultsEditor';
 	protected _rawOptions: BareResultsGridInfo;
 
-	private resultsView: QueryResultsView;
-	private styleSheet = DOM.createStyleSheet();
+	private resultsView?: QueryResultsView;
+	private styleSheet?= DOM.createStyleSheet();
```

Comment:
So I'd suggest that you make this non-optional and then in dispose instead of setting it to undefined just remove it. That accomplishes the actual important thing which is to remove the styles from the DOM and then makes all these undefined checks unnecessary.

You could also make a getOrCreateStyleSheet helper method that will create the style sheet if it doesn't exist - but I think for purposes here my first suggestion is cleanest.